llmfoundry/data/finetuning/collator.py

```diff
@@ -106,7 +106,7 @@ def __init__(
 
     def __call__(self, examples: List[Dict[str,
                                            Any]]) -> Dict[str, torch.Tensor]:
-        for check_key in ['input_ids', 'labels', 'attention_mask']:
+        for check_key in ['input_ids', 'labels']:
```

Comment:
whats going on here? was this just a mistake before to require this key? seems like yes

Reply:
yeah, it looks it was a mistake. Both `self._process_and_batch_decoder_only` and `self._process_and_batch_encoder_decoder` will compute and add `attention_mask` to the sample. So I think it's not needed here.

Reply:
👍